src/aleph_vrf/settings.py

```diff
@@ -30,6 +32,10 @@ class Settings(BaseSettings):
         default=None, description="Application private key to post to aleph.im."
     )
 
+    @property
+    def api_host(self) -> HttpUrl:
+        return self.API_HOST or sdk_settings.api_host
```

Comment:
```suggestion
        return self.API_HOST or sdk_settings.API_HOST
```
I think the SDK don't have the api_host property, like I just checked here: https://github.com/aleph-im/aleph-sdk-python/blob/main/src/aleph/sdk/conf.py#L25